tools/gcp-broker-provider/bin/run.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Following script provisions and deprovisions GKE cluster.
+#
+# Expected environment variables:
```

Comment:
I assume those envs are needed? If yes, please say something more in README.md:
- Prerequisites - google setup
- Usage: how to run checker.sh and run.sh (I understand those script are executed via bundle/JOb)